pkg/network/ebpf/c/protocols/kafka/kafka-parsing.h

```diff
@@ -1804,4 +1816,26 @@ static __always_inline void update_topic_name_size_telemetry(kafka_telemetry_t *
     __sync_fetch_and_add(&kafka_tel->topic_name_size_buckets[bucket_idx], 1);
 }
 
+// update_classified_fetch_api_version_hits_telemetry updates the array keeping track of fetch api versions hits in the telemetry
+static __always_inline void update_classified_fetch_api_version_hits_telemetry(kafka_telemetry_t *kafka_tel, __u8 api_version) {
+    __u8 bucket_idx = api_version - 1;
```

Comment:
fetch can have api_version of `0` (produce as well). so why do we need the `-1`?